ext/js/pages/settings/profile-controller.js

```diff
@@ -549,7 +549,8 @@ export class ProfileController {
             Number.isFinite(intValue) &&
             intValue >= 0 &&
             intValue < this.profileCount ?
-            intValue : null
+            intValue :
```

Comment:
Personally, forcing these to line break seems a bit overkill to me

Reply:
The reason I made this change was to just enforce consistency, but I can remove it if you feel strongly about it. There are only 9 situations in the codebase where a conflicting pattern was used, and about ~40 places where a 3-line ternary was used.

Here is an example of something that should definitely never be a single line:

https://github.com/themoeway/yomitan/blob/805cf9cb3ea744a6d7c0a5da27fc9ef4e6f08626/ext/js/app/popup.js#L807-L811

So the intent was to just try to make sure things like that cannot be made into a single line.